tests/test_handlers.py

```diff
@@ -456,6 +456,34 @@ def test_am1bcc_parameterization():
     # assert vjp_fn(charges_adjoints) == None
 
 
+def test_resp_parameterization():
```

Comment:
We should add a test for determinism (either in the presence or absence of a seed) since the underlying algorithm uses a stochastic conformer generator. i.e. we want to make sure that calling RESP multiple items returns identical charges on *fresh* non-cached molecules.